packages/base/src/styling/ThemingParameters.ts

```diff
@@ -375,17 +572,29 @@ export const ThemingParameters = {
   sapInfobar_Background: 'var(--sapInfobar_Background)',
   sapInfobar_Hover_Background: 'var(--sapInfobar_Hover_Background)',
   sapInfobar_Active_Background: 'var(--sapInfobar_Active_Background)',
+  sapInfobar_NonInteractive_Background: 'var(--sapInfobar_NonInteractive_Background)',
+  sapInfobar_TextColor: 'var(--sapInfobar_TextColor)',
   sapObjectHeader_Background: 'var(--sapObjectHeader_Background)',
   sapObjectHeader_BorderColor: 'var(--sapObjectHeader_BorderColor)',
+  sapObjectHeader_Hover_Background: 'var(--sapObjectHeader_Hover_Background)',
+  sapObjectHeader_Title_TextColor: 'var(--sapObjectHeader_Title_TextColor)',
+  sapObjectHeader_Title_FontSize: 'var(--sapObjectHeader_Title_FontSize)',
+  sapObjectHeader_Title_SnappedFontSize: 'var(--sapObjectHeader_Title_SnappedFontSize)',
+  sapObjectHeader_Title_FontFamily: 'var(--sapObjectHeader_Title_FontFamily)',
+  sapObjectHeader_Subtitle_TextColor: 'var(--sapObjectHeader_Subtitle_TextColor)',
```

Comment:
Those variables might be interesting for our DynamicPage/ObjectPage

Reply:
I replaced the placeholder hover colors with the correct ones, we still need to check the title and subtitle styles though